container_files/assets/www/doc/builtin/functions/EM.md

```diff
@@ -0,0 +1,40 @@
+# EM Function
```

Comment:
My reasoning was that we could in time support more statistical models than just Gaussian Mixture.